packages/components/dialog/src/dialog.ts

```diff
@@ -94,6 +97,14 @@ export class Dialog extends ScopedElementsMixin(LitElement) {
     super.connectedCallback();
 
     this.inert = true;
+
+    this.#focusTrapController = new FocusTrapController(this, { disableCancel: this.disableCancel });
```

Comment:
Is this necessary in `connectedCallback` perse? Can't you initialize the controller on line 69?

Reply:
You probably also want to update `disableCancel` when that property changes? So add that code to `updated()` lifecycle callback?